tests-manual/gpu/CMakeLists.txt

```
@@ -9,17 +9,14 @@ setup_memory_debugger()
 setup_thread_debugger()
 set_target_properties(${TARGET_NAME} PROPERTIES FOLDER "Tests/manual-tests/")
 link_hifi_libraries(
-    shared task networking gl 
-    ktx shaders gpu procedural octree image 
-    graphics model-networking fbx hfm animation material-networking
-    script-engine render render-utils 
-    ${PLATFORM_GL_BACKEND}
+        shared task networking gl
+        ktx shaders gpu procedural octree image
+        material-networking ktx shaders model-networking
+        graphics model-networking model-serializers hfm animation material-networking
+        script-engine render render-utils
+        ${PLATFORM_GL_BACKEND}
```

Any reason we are adding double indentation here?
```suggestion
    shared task networking gl
    ktx shaders gpu procedural octree image
    material-networking ktx shaders model-networking
    graphics model-networking model-serializers hfm animation material-networking
    script-engine render render-utils
    ${PLATFORM_GL_BACKEND}
```